lib/vars.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const url = require('url')
+
+class Vars {
+  constructor (env) {
+    this.env = env
+  }
+
+  get host () { return this.env.HEROKU_HOST || 'heroku.com' }
+  get apiHost () { return this.host.startsWith('http') ? this.host : `api.${this.host}` }
```

Comment:
Why can it start with "http"? I don't think that will work with the .netrc format when it tries to fetch the token.